meteor/client/ui/Shelf/ShelfDashboardLayout.tsx

```diff
@@ -265,3 +259,41 @@ export function ShelfDashboardLayout(props: IShelfDashboardLayoutProps) {
 		</div>
 	)
 }
+
+export interface InterpolatedPropsSource {
```

Comment:
I agree with the first part, but I disagree with reusing an existing interface for studio and show style or renaming this one. This is its own interface for a reason. It's a part of `interpolatePanelStrings`' contract. I actually considered adding `Playlist` to this interface as well, because hypothetically someone may want to use the name of a playlist or some of its metadata. With what you're proposing, sucha addition would require this interface to be renamed, which would potentially affect every place where `interpolatePanelStrings` is used, and where that interface is imported. Why do that if you can have a name that reflects the purpose?